db/migrate/20211130152323_create_branch_products.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBranchProducts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :branch_products do |t|
+      t.references :branch
+      t.references :product
+      t.integer :price_in_cents
+      t.string :status, default: 'available'
```

Comment:
either change this to enum or a boolean if only two values are allowed

Reply:
Fixed.